public/js/twiliochat.js

```diff
@@ -90,12 +90,15 @@ var twiliochat = (function() {
   function connectMessagingClient(tokenResponse) {
```

Comment:
What about just passing the actual token instead of the object which contains it?